<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="CbWebinars">
    <meta name="author" content="Areejit Samal (asamal@imsc.res.in)">
    <meta name="generator" content="HTML PHP MYSQL CSS JS">
    <title>
    Computational Biology Webinars
    </title>
    
    <script src="http://code.jquery.com/jquery-latest.js"></script>
    <script src="js/bootstrap.bundle.min.js"></script>
    <script src="js/popper.min.js"></script>
    
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
    
    <style>
	.nav-pills .nav-link.active, .nav-pills .show>.nav-link { color: #fff; background-color: #d0836e90; font-weight: 600; }

	ul.nav { font-weight: 600; }

	ul.nav a:hover { color: grey; font-weight: 600; }

	body {
	    font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
	    font-size: 1rem;
	    font-weight: 400;
	    line-height: 1.5;
	    color: #212529;
	    text-align: left;
	    background-color: #f7ede4;
	}

	a {
	    color: #d0836e;
	    text-decoration: none;
	    background-color: transparent;
	}

	div {
	  text-align: justify;
	  text-justify: inter-word;
	}

	.section-head { background-color: #d0836e90; }


	#myBtn {
	  display: none;
	  position: fixed;
	  bottom: 20px;
	  right: 30px;
	  z-index: 99;
	  font-size: 18px;
	  border: none;
	  outline: none;
	  background-color: #d0836e;
	  color: white;
	  cursor: pointer;
	  padding: 15px;
	  border-radius: 4px;
	}

	#myBtn:hover {
	  background-color: grey;
	}

    </style>
    
</head>

<body>
<div class="section-head container-fluid">
<div class="container pl-5 pt-3">
<center>
<b><h3><font color="#fff">Computational Biology Webinars</font></h3></b>
<img src="images/imsclogo.png" height="9%" width="9%">
<h5><font color="#fff">The Institute of Mathematical Sciences (IMSc)
<br>Chennai, India
</font></h5>
</center>
</div><br>
</div>


<!-- container section: start-->
<div class="container px-5 pt-3">
<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>

<ul class="nav nav-pills nav-fill">
  <li class="nav-item">
    <a class="nav-link active" href="#July2020">July 2020</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#August2020">August 2020</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#September2020">September 2020</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#October2020">October 2020</a>
  </li>
  <li class="nav-item">
    <a class="nav-link" href="#November2020">November 2020</a>
  </li>
   <li class="nav-item">
    <a class="nav-link" href="#December2020">December 2020</a>
  </li>
</ul>
<br>

<div id="July2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Computational Gastronomy: Leveraging Artificial Intelligence for Data-driven Food Innovations</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/GaneshBagler_July30_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/GaneshBagler_July30_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://faculty.iiitd.ac.in/~bagler/" target="_blank">Ganesh Bagler</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indraprastha Institute of Information Technology (IIIT), Delhi<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
July 30, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Cooking forms the core of our cultural identity other than being the basis of nutrition and health. The increasing availability of culinary data and the advent of computational methods for their scrutiny is dramatically changing the artistic outlook towards gastronomy. Starting with a seemingly simple question, ‘Why do we eat what we eat?’ data-driven research conducted in our lab has led to interesting explorations of traditional recipes, their flavor composition, and health associations. Our investigations have revealed ‘culinary fingerprints’ of regional cuisines across the world, starting withthe case study of Indiancuisine. Application of data-driven strategies for investigating the gastronomic data has opened up exciting avenues giving rise to an all-new field of ‘Computational Gastronomy’. This emerging interdisciplinary science asks questions of culinary origin to seek their answers via the compilation of culinary data and their analysis using methods of statistics, computer science, and artificial intelligence. Along with complementary experimental studies, these endeavors have the potential to transform the food landscape by effectively leveraging data-driven food innovations for better health andnutrition</font></h6></p>
	</div>
</div>
</div><br>

</div>

<div id="August2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Efficient mapping of microbial metabolic systems by integrating Genomics with 13C Fluxomics</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/ShyamMasakapalli_Aug4_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/ShyamMasakapalli_Aug4_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://faculty.iitmandi.ac.in/~shyam/" target="_blank">Shyam K. Masakapalli</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institute of Technology (IIT), Mandi<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 4, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
13C fluxomics integrated with other -omics provide Genotype to Phenotype predictive framework and unprecedented insights into the cellular systems. This talk will mainly focus on strategies we adopt in efficient integration of comparative genomics and 13C tracer studies to decode the metabolic phenotypes of microbial metabolic systems. First, the standardized workflow leading to efficient metabolic mapping and flux maps of bacterial systems will be introduced. Secondly, our latest work ondeciphering metabolic phenotypes of a phytopathogen <em>Ralstonia solanacearum</em> using parallel 13C tracer feeding and metabolic modelling will be presented. The study highlights how the metabolic models and flux maps have relevance to agriculture and industrial bioprocessing. Finally, the bottlenecks and potential opportunities in deciphering Plant-microbial metabolic cross talk using stable isotopes will be discussed.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">The chemical space and its three statistical regimes</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/GuillermoRestrepo_Aug6_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/GuillermoRestrepo_Aug6_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/site/guillermorestrepo/Home" target="_blank">Guillermo Restrepo</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 6, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Exploring the chemical space, spanned by all chemical species, is at the core of chemical activity. Records of how the space is expanded through chemical reactions exist in more than 200 years of scientific publications, now available in electronic databases. Despite its importance, very little is known about this exploration throughout history and about the driving forces affecting it. Here we show, by analysing millions of reactions stored in Reaxys® database, that the exploration is marked by three regimes and by social and scientific factors. The first regime was dominated by volatile inorganic production and ended about 1860, when structural theory gave place to a century of guided production, the organic regime. After 1980 began the least volatile regime, the current organometallic one. We found a stable 4.4% annual growth rate of production of new compounds not affected in the long run neither by World Wars nor by the introduction of new theories. However, World Wars have delayed production. Moreover, we found that chemists have been conservative in the selection of their starting materials but have beenhistorically motivated tounveilnew compounds of the space.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Structural and dynamical insights into mammalian circadian clock proteins</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/AshutoshSrivastava_Aug11_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/AshutoshSrivastava_Aug11_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://iitgn.ac.in/faculty/bioe/fac-ashutosh" target="_blank">Ashutosh Srivastava</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Institute of Transformative Bio-Molecules, Nagoya University, Japan<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 11, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The physiology and behavior of almost all living organisms on earth is synchronized to a 24-hour solar cycle by a well-regulated molecular clock mechanism. This internal biological clock regulates a host of cellular responses to the environment, ranging from gene expression and cell division in cyanobacteria, to photosynthesis in plants and finally to the sleep/wake cycles in mammals (commonly referred as circadian rhythms). In this talk, I will present the work that we have been doing to not only enhance the understanding of molecular mechanisms regulating circadian clock [1] but also to develop therapeutic interventions to modulate the circadian rhythms in mammals [2, 3]. Using hybrid/integrative modeling, involving multiple experimental and computational methods, we have been able provide mechanistic insights into the role of cryptochromes–a core clock protein, in regulating circadian period length, thus directly relating protein structure and dynamics to in vitro and in vivo experimental observations [1].<br><br>

<ol>
<li>Jennifer Fribourgh*, Ashutosh Srivastava*, Colby Sandate* et al. (2020); Dynamics at the serine loop underlie differential affinity of cryptochromes for CLOCK:BMAL1 to control circadian timing; eLife; 9:e55275 (*Equal Contribution)</li>
<li>Tsuyoshi Oshima, Yoshimi Niwa, Keiko Kuwata, Ashutosh Srivastava, et al.(2019); Cell-based screen identifies a new potent and highly selective CK2 inhibitor for modulation of circadian rhythms and cancer cell growth; Science Advances, 5, 1, eaau9060</li>
<li>Simon Miller, You Lee Son, Yoshiki Aikawa, Eri Makino, Yoshiko Nagai, Ashutosh Srivastava, et al. (2020); Isoform-selective regulation of mammalian cryptochromes. Nature Chemical Biology 16, 676-685.</li>
</ol>
</font></h6></p>
	</div>
</div>
</div><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Cell‐specific responses to the cytokine TGF β are determined by variability in protein level</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/UddipanSarma_Aug13_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/UddipanSarma_Aug13_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://in.linkedin.com/in/uddipan-sarma-4068456b" target="_blank">Uddipan Sarma</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Vantage Research, Chennai<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 13, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Information encoded in the dynamics of signaling pathways often elicit critical cell fate decisions. For instance, sustained dynamics of TGFβ pathway impart growth inhibition, a property abrogated in diseases like cancer. To understand how cells encode the extracellular input and transmit its information to elicit appropriate responses, we acquired quantitative time‐resolved measurements of pathway activation at the single‐cell level. We compared the signaling dynamics of thousands of individual cells and build mathematical models to understand the regulatory processes controlling the cell specific dynamics, both sustained and transient. Our combined experimental and theoretical study revealed that the response to a given dose of TGFβ is determined specifically by the levels of defined signaling proteins in individual cells. Heterogeneity in signaling protein expression led to decomposition of cells into classes with qualitatively distinct signaling dynamics and corresponding phenotypic outcome. Also, negative feedback regulators promote heterogeneous TGFβ signaling, as SMAD7 (a negative regulator of the pathway) knock‐out specifically affected the signal duration in a subpopulation of genetically identical cells. Taken together, our study established a quantitative framework that allows predicting and testing sources of cellular signaling heterogeneity.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Unravelling rice cellular physiology</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/SudipKundu_Aug18_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/SudipKundu_Aug18_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.caluniv.ac.in/academic/department/Bio-Physics/Sudip-K.pdf" target="_blank">Sudip Kundu</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
University of Calcutta, Kolkata<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 18, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
More than 20% caloric intake of the whole world population comes from rice; however, this rice production is under several biotic and abiotic stresses. Thus, society needs efficient stress tolerant high yield rice cultivars. A deep understanding of the rice cellular and plant physiology, and how its outcome depends on the interactions of several levels of different cellular networks will help the rice biotechnologist to achieve this goal. Towards this aim, the focus of this presentation would be use of analytical techniques to understand the rice cellular physiology. Firstly, I will discuss how integration of different omics data and network theory can not only unravel various regulatory interactions connecting phenotypic changes with cellular and/or molecular events triggered by stress, but also provides a framework to deepen our understanding of stress cellular physiology. However, this technique will not be very useful to understand the cellular metabolism and thus, we use two different metabolic modelling tools, namely flux balance analysis (FBA) and elementary flux mode (EFM) analysis techniques. Secondly, I will describe how we create a structural metabolic model that contains the reactions that participate in photorespiration in the plastid, peroxisome, mitochondrion and cytosol, and the metabolite exchanges between them, and analyse this model (i) to understand biochemical basis of leaf ammonium accumulation and chlorosis in GS2 mutant type and (ii) to address the impact of photorespiration on metabolism. We also provide a formal demonstration that photorespiration itself does not impact on the CO2:O2 ratio (assimilation quotient), except in those modes associated with concomitant nitrate reduction.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Hacking God’s Own Program with Synthetic Genetic Circuits: Artificial neural networks to microgravity sensors in living bacteria</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/SangramBagh_Aug20_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/SangramBagh_Aug20_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/site/sbaghsyntheticbiology/" target="_blank">Sangram Bagh</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Saha Institute of Nuclear Physics (SINP), Kolkata<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 20, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The molecular connectivity between genes and proteins inside a cell shows a good degree of resemblance with complex electrical circuits. This inspires the possibility of engineering a cell similar to an engineering device. In this talk, we discuss our recent effort to hardware implement artificial neural network (ANN) with engineered bacteria. The abstract mathematical rules of artificial neural network (ANN) are implemented through software, various material based neuromorphic chips, photonics and in-vitro DNA computation. Here we demonstrate the physical realization of ANN in living bacterial cells. We created a single layer ANN using engineered bacteria, where a single bacterium works as an artificial neuron and demonstrated complex chemical information processing with decoders and encoders. To our knowledge, this is the first ANN created by artificial bacterial neurons. Thus, it may have significance creating engineered biological cells as ANN enabled hardware. On the other hand, synthetically engineered microbial cells have numerous projected applications in space bioengineering. Microgravity is a unique property of space. Biological solutions to space travel must consider microgravity as an important component. Here we have created the first biological or biochemical or molecular microgravity sensor in <em>Escherichia coli</em> applying synthetic gene circuits.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Roadmaps of interaction between the coding and the noncoding RNA world: Orchestrating disease biology</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/ZhumurGhosh_Aug25_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/ZhumurGhosh_Aug25_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://www.jcbose.ac.in/faculty-details/zhumur-ghosh" target="_blank">Zhumur Ghosh</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Bose Institute, Kolkata<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 25, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The causal factor that makes cancer easy to recur and metastasize is still an enigma to the scientific community. A subpopulation of cells named as cancer stem cells, within a tumor possess the capacity to self-renew and produce heterogeneous lineages of cancer cells that comprise the tumor. We are primarily interested in unearthing the molecular mechanisms and find the main players that deliver the stem cell its oncogenic potential. In our search for regulators, we came across non coding RNAs, which, once considered as mere junk have emerged to be one of the key players in maintaining both healthy and diseased condition of our system. Comprising of more than 98% of the human genome, they can be categorized into two types viz. small non coding RNAs like microRNAs (miRNAs), piwi interacting RNAs (piRNAs) and long non coding RNAs (lncRNAs). Their versatile regulatory role motivated us to delve deeper into their disposition and action, and all detailed information of our findings and analysis are now maintained in the form of two dedicated databases, LncRBase and piRNAQuest. Our interest also lies in their interacting coding RNA partners, the culmination of which is our noncoding RNA target prediction tools-miRTPred and LncRTPred, and we are working forward to bring about similar solutions for piRNAs in the future. Placing together all these information and connecting the dots, we hope to unravel some of the hidden roadmaps of interaction between the coding and the noncoding RNA world.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Modelling Approaches to understand the challenges of Cancer Metabolism</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/RamRupSarkar_Aug27_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/RamRupSarkar_Aug27_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://academic.ncl.res.in/rr.sarkar" target="_blank">Ram Rup Sarkar</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
CSIR-National Chemical Laboratory, Pune<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
August 27, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
"Cancer cells exhibit characteristic phenotypic plasticity that allows adaptive cellular reprogramming facilitating rapid proliferation, evading immunosurveillance and survival under stress. Cancer metabolism, an emerging hallmark of cancer cells, is one such adaptation that exhibit distinctive phenotypic changes and have been considered as signatures for different cancer cells. Metabolites can directly influence stress response pathways, chromatin modifications and gene expression that collectively drive tumor development. We will be discussing about the metabolic complexities in cancer with reference to a particular type of brain cancers known as glioblastomas. Metabolic alterations like the Warburg effect, Glutaminolysis, etc., help glioblastomas to survive stringent conditions. However, it is difficult to design a holistic experimental setup that could capture multiple pathways simultaneously. In recent years, this limitation is largely being handled by computational and mathematical biology study of large-scale comprehensive signaling and metabolic networks. In this lecture, we will discuss about the two broadly classified computational techniques to address this biological problem: (i)Steady-state modelling approach and (ii) Dynamic modelling approach. In the first part of the lecture, we will discuss about a popularly used steady state approach known as Constraint-Based Metabolic Modelling. This approach makes use of linear optimization to formulate the cancer metabolic network in mathematical form. The technique provides a holistic perspective of the pathway behavior and changes in a context specific metabolic network of glioblastoma. A network consisting of 13 pathways including Glycolysis, TCA, Oxidative phosphorylation, Glycineserine metabolism, Cysteine metabolism and Glutamate metabolism pathways was reconstructed [1]. The model was used to interpret biological questions like the differences in pathways during a normal and a glioblastoma scenario, essential metabolites for glioblastoma growth and combinations of metabolic reactions that could be used for treatment or as drug targets. The pathways were observed to be re-routed towards glutathione pathway, which is the anti-oxidant machinery of the cell. Essentiality analysis displayed that cystine and glucose were essential for glioblastoma growth in the given context. The combination of glycine-serine pathway enzymes was highlighted as combinatorial therapeutic targets. In the second part of the lecture, we will discuss about dynamic modelling approach using ordinary differential equation (ODE). This approach requires the detailed understanding of the biological system and the knowledge of parameter values like concentration, rate kinetics, etc. We have used this approach to build an ODE model for glioblastoma to understand the effect of changing concentration of Reactive Oxygen Species (ROS) in determining the pro-apoptotic or anti-apoptotic fate of gliomas [2]. The model consists of a smaller subset of metabolic pathways that were considered in the constraint-based model, which are relevant to the anti-oxidant machinery. A total of 25 rate equations with Michaelis-Menten and modified Michaelis-Menten equations were formulated, that consisted of 35 variables and 123 parameters. Analysis of the model show that the regulation of certain parameters along with the thiol (GSH/GSSG) and redox (NADPH/NADP+) ratio could determine the dual behavior of ROS in gliomas.<br><br>

<ol>
<li>Bhowmick, R., Subramanian, A., & Sarkar, R. R. (2015). Exploring the differences in metabolic behavior of astrocyte and glioblastoma: A flux balance analysis approach. Systems and Synthetic Biology, 9, 159 - 177, <a href="https://doi.org/10.1007/s11693-015-9183-9">DOI:10.1007/s11693-015-9183-9</a>.</li>
<li>Bhowmick, R., & Sarkar, R. R. (2020). Differential suitability of reactive oxygen species and the role of glutathione in regulating paradoxical behavior in gliomas: A mathematical perspective. PloSOne, 15(6), e0235204. <a href="https://doi.org/10.1371/journal.pone.0235204">DOI:10.1371/journal.pone.0235204</a>.</li>
</ol>
</font></h6></p>
	</div>
</div>
</div><br>

</div>

<div id="September2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Systems Biology of Metabolism for Microbial Cell Factories</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/AmitGhosh_Sep1_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/AmitGhosh_Sep1_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://www.energy.iitkgp.ac.in/~amitghosh/Research.html" target="_blank">Amit Ghosh</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institute of Technology (IIT), Kharagpur<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
September 1, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Microbial metabolism can be harnessed to convert sugars and other carbonaceous feedstocks into a variety of chemicals, fuels, and drugs. Cellular Metabolism is very complex involving thousands of reactions and metabolites. To understand the complexity of metabolism, mathematical models were developed for holistic studies. The most popular mathematical model in biology is referred to as Genome-scale Metabolic Model (GEM). GEMs are expanding our understanding of cellular metabolism and its dynamics. It supplies us with a systems level picture of the metabolism derived from its genotype. Rational design of metabolism is very important for production of bioproducts. In silico prediction of metabolic flux distribution of the metabolic pathways enabled us to decide the time consuming steps in metabolic engineering. Metabolic engineering involves improvement of bio-chemicals formation through the modification of specific genes or addition of new genes involved in biochemical reactions with the use of genome engineering tools such as CRISPR/Cas9. We will discuss the use of “omics”-driven tools of modern systems biology for microbial production of bioproducts and advanced biofuels.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Chaperoning four billion years of protein evolution</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/SauravMallik_Sep8_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/SauravMallik_Sep8_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://in.linkedin.com/in/saurav-mallik-3a925a75" target="_blank">Saurav Mallik</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Weizmann Institute of Technology, Israel<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
September 8, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Along evolutionary time, starting from simple a beginning, the repertoire of life’s proteins has widely expanded. A systematic analysis across the Tree of Life (ToL) depicts that from simplest archaea to mammals, the total number of proteins per proteome expanded ~200-fold. In parallel, proteins became more complex: protein length increased ~3-fold, and multi-domain proteins expanded ~300-fold. Apart from duplication and divergence of existing proteins, expansion was driven by birth of completely new proteins. Along the ToL, the number of different folds expanded ~10-fold, and fold-combinations ~40-fold. Proteins prone to misfolding and aggregation, such as repeat and beta-rich proteins, proliferated ~600-fold. To control the quality of these exponentially expanding proteomes, core-chaperones, ranging from HSP20s that prevent aggregation, to HSP60, HSP70, HSP90 and HSP100 acting as ATP-fueled unfolding and refolding machines, also evolved. However, these core-chaperones were already available in prokaryotes ~3 billion years ago, and expanded linearly, as they comprise ~0.3% of all genes from archaea to mammals. This challenge—roughly the same number of core-chaperones supporting an exponential expansion of proteome complexity, was met by: (i) higher cellular abundances of the ancient generalist core-chaperones, and (ii) continuous emergence of new substrate-binding and nucleotide-exchange factor co-chaperones that function cooperatively with core-chaperones, as a network.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Detection and characterization of rare transcriptomes</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/DebarkaSengupta_Sep15_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/DebarkaSengupta_Sep15_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.iiitd.ac.in/debarka" target="_blank">Debarka Sengupta</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indraprastha Institute of Information Technology (IIIT), Delhi<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
September 15, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Over the past decade, single-cell transcriptomics has transcended our understanding of complex biological systems. Single-cell expression readouts allow precise characterization of cellular heterogeneity in normal as well as diseased tissues. Further, single-cell RNA sequencing allows the detection of cells, present in a minority within a tissue. Examples of rare cell types include circulating tumor cells, cancer stem cells, circulating endothelial cells, endothelial progenitor cells, antigenspecific T cells, invariant natural killer T cells, etc. In practical scenarios, rare cell detection poses significant methodological challenges. In this talk, I will discuss our group's journey of raising a new breed of computational techniques for the efficient detection of rare transcriptomes.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Machine Learning for Engineering Biology in the Era of Network Science</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/ShailzaSingh_Sep22_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/ShailzaSingh_Sep22_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://in.linkedin.com/in/shailza-singh-b845225a" target="_blank">Shailaza Singh</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
National Centre for Cell Science (NCCS), Pune<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
September 22, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Network biology is an investigative and constructive means of understanding the complexities of biology. Substantial progress in the field has resulted in the creation of systems driven synthetic gene circuits which can be used for a tunable response in a cell. These tunable elements can be applied to treat diseased conditions for a transition to a healthy state. Though in its nascent stage of development synthetic biology is beginning to use its constructs to bring engineering approaches using machine learning into biomedicine for treatment of infectious disease. Many engineered peptides/proteins have made their way to therapeutics and diagnostics. I will discuss few success stories from our lab in metabolic, signaling and transcriptional regulatory network wherein we have been able to modulate the gene expression from an anti-inflammatory to a pro-inflammatory phenotype in in vitro <em>L. major</em> infected macrophage model.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Percolation in Planar Cell Polarity</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/BiplabBose_Sep24_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/BiplabBose_Sep24_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.iitg.ac.in/biplabbose/" target="_blank">Biplab Bose</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institute of Technology (IIT), Guwahati<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
September 24, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Epithelial cells, like those on our skin or in the wings of a fly, are polarized, with an asymmetric distribution of molecules and structures inside a cell. In Planar Cell Polarity (PCP), all epithelial cells in a particular region get polarized along the proximal-distal axis. PCP is an example of self-organization through local and global interactions between cells. Inspired by the similarity between PCP and other self-organizing phenomena, we have used a lattice based spin model for PCP that mimics the alignment of cells through local interactions. In this model, the segregation of protein-complexes within a cell is equivalent to spin-exchange, and interaction between neighboring cells through protein complexes is equivalent to spin-spin interactions. We investigated the equilibrium behavior of this model. In this model, the alignment of cells leads to the formation of clusters of aligned cells, and such clustering exhibits a percolation transition. Even though the alignment of a cell in this model depends upon its neighbors, finite-size scaling analysis shows that this model belongs to the universality class of simple 2-dimensional random percolation. In this webinar, I will discuss various behaviors of this toy model and would try to illustrate that if we remove the bells and whistles, many complicated biological phenomena can be equated with simple physical models.</font></h6></p>
	</div>
</div>
</div><br>

</div>

<div id="October2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Shapeshifters in cancer: how do tumor cells switch among different phenotypes to drive aggressive behavior</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/MohitJolly_Oct1_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/MohitJolly_Oct1_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://be.iisc.ac.in/~mkjolly/" target="_blank">Mohit K. Jolly</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institute of Science (IISc), Bengaluru<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 1, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
"Metastasis (the spread of cancer cells from one organ to another) and therapy resistance cause above 90% of all cancer-related deaths. Despite extensive ongoing efforts in cancer genomics, no unique genetic or mutational signature has emerged for metastasis. However, a hallmark that has been observed in metastasis is adaptability or phenotypic plasticity – the ability of a cell to reversibly switch among different phenotypes in response to various internal or external stimuli. Phenotypic plasticity has also been recently implicated in enabling the emergence of resistance for many cancers across multiple therapies. However, a mechanistic understanding of these processes from a dynamical systems perspective remains incomplete. This talk will describe how mechanism-based mathematical models for phenotypic plasticity can enable our improved understanding of cellular decision-making at individual and population levels from these perspectives: 
a) Multistability (how many cell states exist en route?) 
b) Reversibility/irreversibility (do cells come across a ‘tipping point’ at specific time and/or dose of inducers beyond which they do not revert?) 
c) Hysteresis (do transitioning cells follow same/different paths?) 
d) Cell-cell communication (how do cells affect tendency of their neighbors to exhibit plasticity?) 
Collectively, our work highlights how an iterative crosstalk between mathematical modeling and experiments can both generate novel insights into the emergent nonlinear dynamics of cellular transitions and uncover previously unknown accelerators of metastasis and therapy resistance.</font></h6></p>
	</div>
</div>
</div><br>"
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Systems biology of small and large systems</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/SuchetaGokhale_Oct6_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/SuchetaGokhale_Oct6_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://in.linkedin.com/in/sucheta-gokhale-943b5416" target="_blank">Sucheta Gokhale</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
TATA Chemicals, Pune<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 6, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Biological processes are regulated at multiple levels ranging from chromatin remodeling for gene expression regulation, to specific localization of enzymes for metabolic regulation. These processes are best understood by analyzing them at different abstraction levels. I will present two studies, one focusing on kinetic modeling of a small system of divergent transcription and another focusing on genome scale metabolic flux balance analysis of gut microorganisms. One of the important factors affecting gene expression is specific promoter arrangement. Divergent promoters are the promoters in which upstream reverse oriented transcription occurs in the regions devoid of annotated genes but shows the presence of core promoter elements. We developed a kinetic model of divergent transcription in order to understand the regulatory effects of the process. We analysed the model using ChIP-seq data and tested the predictions about expression levels using nascent RNA GRO-seq and CAGE data. Moving from a small to whole genome metabolic modeling, I will present our work focusing on analyzing gut microbial metabolic models for post-biotic production abilities. Gut microbiome is now considered as a new organ system in our body. It is known to produce a variety of useful metabolites. We reformed and analysed models of gut microorganisms for their response to various prebiotics and oligosaccharides for post-biotic production. As one of the use-case, we focused on flux redistribution that can lead to vitamin-D precursor synthesis, in microbe-human joint metabolic model.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Stochastic dynamics in low dimensional systems</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/RKBrojenSingh_Sep10_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/RKBrojenSingh_Sep10_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://ccbb.jnu.ac.in/brojen/" target="_blank">R.K. Brojen Singh</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Jawaharlal Nehru University (JNU), New Delhi<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 9, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The processes of the systems far below thermodynamics limit generally exhibit stochastic nature, and the evolved noise becomes one important parameter which regulate the systems’ dynamics. Complex systems are multi-dimensional systems in general and study such systems are quite difficult both analytically as well as numerically. However, dimensional reduction of such systems with suitable conditions allow us to study important behavior of the systems with significant accuracy. We would like introduce stochastic formalism of complex systems in general. Then techniques to reduce the systems’ dimension to solvable one with examples and solving techniques will be explained with examples. The observation of various distinct noise driven patterns, which may correspond to various systems’ states, will be explained.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Genomes - from Personal to Populations and back</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/VinodScaria_Oct15_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/VinodScaria_Oct15_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://vinodscaria.rnabiology.org/" target="_blank">Vinod Scaria</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Institute of Genomics and Integrative Biology (IGIB), Delhi<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 15, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
One of the major advances that happened in the last decade has been the availability of fast, efficient and cost-effective DNA sequencing which has seen tremendous applications in healthcare and research. This has been largely possible with the advent of newer sequencing approaches which offer a higher throughput, lower cost and faster turnaround times, and appropriate computational tools to enable the handling, processing and mining of information from them. On one end, this technology enables to elucidate the genomes of individuals and how genomic variations could influence the life of the individual, while on the other side, it enables one to peek into the pathophysiology of diseases and have a mechanistic view of the disease processes. It is widely believed that these advances would be immensely useful in healthcare through Predictive, Preventive, Precise, Personalised and Participatory Medicine. In the present talk, we would describe our experiences with personal genome sequencing and use of genomics approaches and model systems towards this end. We have employed a number of novel approaches towards harnessing the BigData challenge including involvement of students towards analysis and model-building.The talk would also discuss the way forward, detailing the ongoing initiatives including the GUaRDIAN Consortium involving a large clinical network towards realising the dream.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">How is information processed in developing spinal cord?</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/MarcinZagorski_Oct20_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/MarcinZagorski_Oct20_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://zagorskigroup.com/" target="_blank">Marcin Zagorski</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Jagiellonian University, Kraków, Poland<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 20, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The development of multicellular organisms is a dynamic process in which cells divide, rearrange, and interpret molecular signals to adopt specific cell fates. Despite the intrinsic stochasticity of cellular events, the cells identify their position within the tissue with striking precision of one cell diameter inthe fruit fly or three cell diameters inthe vertebrate spinal cord. How do cells acquire this positional information? How is this information encoded and how do cells decode it to achieve the observed level of cell fate reproducibility? These are fundamental questions in biology that are still poorly understood. In this talk, I will combine both information theory methods and mechanistic models to address these questions in the context of spinal cord development. I will consider the two opposing morphogen signals that are integrated to specify the arrayed pattern of neural progenitor domains that later on give raise to different type of neurons. Based on the maximum likelihood estimation rule I will define decoding map that provides predictions for shifts in the target gene domains in mutants. The predictions will be validated using experimental data obtained from naïve chick neural plate explants and from embryos with altered ventral morphogen signaling. I will present a simple model of a gene regulatory network that integrates the two morphogen signals and is sufficient to recapitulate the observed shifts in the target domains. I will investigate to what extent the level of noise in the input signals affects precision of the resulting gene expression pattern. In the long-term, the contribution from the proposed basic research might be utilized in designing neuroregenerative therapies.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Why cousins are more similar than mother-daughters: implications for cell cycle regulation</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/ShaonChakrabarti_Oct22_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/ShaonChakrabarti_Oct22_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.ncbs.res.in/faculty/shaon" target="_blank">Shaon Chakrabarti</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
National Centre for Biological Sciences (NCBS), Bengaluru<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 22, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Recent developments in microscopy techniques have allowed probing of cellular dynamics at an unprecedented resolution and throughput. For example, these advances are now allowing us to study the phenomenon of cellular proliferation at the single cell level, rather than the population dynamics of millions of cells. However, interpreting the inevitably noisy datasets associated with such single cell measurements is a fundamental challenge and provides an exciting opportunity for developing physical models in combination with statistical inference. Here I will present work where we combined time-lapse microscopy and Bayesian inference to uncover surprising correlations in the division and death times of colon cancer cells closely related by lineage, both before and during chemotherapy treatment. These correlations could not be explained using simple protein production degradation models that are currently believed to underlie cell fate control. We then developed a stochastic model explaining how the observed correlations can arise from oscillatory mechanisms underlying cell cycle control. Our model was able to recapitulate the data only with specific oscillation periods that fit measured circadian rhythms, suggesting that cell to cell heterogeneity in cell cycle progression rates may arise from circadian control over the cell cycle. Finally, I will discuss some new experiments and theory we are developing to further investigate the role of the circadian clock in cellularp roliferation, both in cancer as well as in stem cells.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Computational tackling of biological systems: Circadian clock and human microbiome</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/Pan-JunKim_Oct27_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/Pan-JunKim_Oct27_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="" target="http://panjunkim.com/">Pan Jun-Kim</font></td></tr>
		<tr><td class="align-top"><font color="grey">
Hong Kong Baptist University, Hong Kong, China<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 27, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
16:00 - 17:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
A primary challenge in biology is to explain how complex phenotypes arise from individual molecules encoded in genes. Molecular interaction networks offer a key to understand how genotypes are translated into phenotypes. For example, sleep/wake cycles in animals are generated by molecular circuits of interacting genes and gene products, called circadian clocks. Circadian clocks are important for both animal and plant life as well. I will discuss how the molecular interactions for rhythmic protein turnover are related to a presumable driving force of the circadian clock machinery—the biosynthetic cost reduction. However, considering only genes in a given organism and its own molecular interactions may not be enough to understand the holistic picture of the organism’s phenotypes. For example, our resident gut microbial community, or gut microbiome, provides us with a variety of biochemical capabilities not encoded in our genes. This human gut microbiome is linked not only to our health, but also to various disorders such as obesity, cancer, and diabetes. We constructed the literature-curated global interaction network of the human gut microbiome mediated by various chemicals. Our network framework shows promise for investigating complex microbe-microbe and host-microbe chemical cross-talk, and identifying disease-associated features.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Reverse modelling metabolic networks</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/AndreaDeMartino_Oct29_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/AndreaDeMartino_Oct29_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://chimera.roma1.infn.it/ANDREA/" target="_blank">Andrea De Martino</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Researcher, CNR Nanotech and Institute of Genomic Medicine, Italy<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
October 29, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The growth performance of microbial populations can be studied by an information theoretic approach relating the mean single-cell growth yield ('fitness') to the entropy of the growth-yield distribution ('information'). Within Mass-Balance models, one finds that, for any value of the information, the achievable fitness is strictly bounded, leading to a theoretical “rate-distortion curve” in the (information,fitness) plane. Next, values of fitness and information for <em>E.coli</em> populations can be inferred from experimental mass spectrometry data probing growth in different conditions. For a large number of experimental datasets, inferred points robustly approach the theoretical bound as the quality of the growth medium improves. Besides giving insight into the interplay between metabolism and gene expression, this approach can yield information that is currently inaccessible by other methods, both insilico and experimental.</font></h6></p>
	</div>
</div>
</div><br>

</div>

<div id="November2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Data-driven approaches to understand and address Antimicrobial Resistance (AMR)</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/AnshuBhardwaj_Nov5_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/AnshuBhardwaj_Nov5_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.imtech.res.in/contact/staff/dr-anshu-bhardwaj/107" target="_blank">Anshu Bhardwaj</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
CSIR-Institute of Microbial Technology, Chandigarh<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
November 5, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Antimicrobial resistance (AMR) is one of the most serious global public health threats as it compromises successful treatment of deadly infectious diseases. The Global Action Plan for AMR (GAP-AMR) has identified better awareness of AMR and strengthening understanding of AMR as two critical verticals. The AB-OpenLab is focussing their research efforts in these verticals of the GAP-AMR. In order to raise awareness on AMR, we have developed a video game on antimicrobial resistance which has an AI bot integrated into the gaming environment as a guide for the players to interact and learn concepts of AMR at their own pace. We will showcase the first version of our game during the talk. We shall then discuss our latest approach to identification of new chemotypes on validated targets as a new lead generation strategy. The talk will discuss some of the recent results on identification of novel chemotypes of mycobacteria and experiments on utilizing collective intelligence for generating resources for drug discovery of mycobacteria and emerging pathogens.</font></h6></p>
	</div>
</div>
</div><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Integrated Systems Approaches to Predict Metabolic Vulnerabilities Of Chemo-resistant Glioblastoma Cells</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/AnuRaghunathan_Nov12_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/AnuRaghunathan_Nov12_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/site/micelabatncl" target="_blank">Anu Raghunathan</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
CSIR-National Chemical Laboratory, Pune<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
November 12, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The field of cancer research is caught in a data deluge by the advent of inexpensive genome-scale high throughput technologies. The complexity of a living system justifies the need for data acquisition at all levels of cell hierarchy from DNA to tissue and organ level delineation. However, just listing candidate genes (from genomic/exome data), metabolite profiles or gene expression signatures (from transcriptomic data) are not enough to understand a complex, multi-hit, multifactorial emergent disease like cancer. Although there are many methods that exist to analyze individual datatypes, no method exists to put heterogenous data-types into a platform or mathematical model and integrate it, let alone predict outcomes and cell behavior. Glioblastoma, the most severe form of brain cancer is even more complex due to its inherent heterogeneity, as the only drug used to treat it is being rendered less useful due to chemo resistance. To understand the difference between cells of glioblastoma that are resistant or susceptible to temozolomide a population of cells from the model cell line U87MG have been isolated and characterized extensively using whole exome sequencing, growth-resistance-metabolic profiling and metabolite respiration phenotyping to understand the intrinsic changes in its molecular components and higher order phenotypes. This talk will discuss these results in the context of a constraints-based tissue specific flux balance model of human metabolism. Such models not only explain the heterogeneity of cells and predict differences in the drug response but can predict alternate targetable cell vulnerabilities. Such scalable work flows could fill a critical need for predictive models for tumor growth and proliferation in personalized medicine.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Structural Analysis of Protein-Protein Interactions</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/RSowdhamini_Nov17_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/RSowdhamini_Nov17_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://www.ncbs.res.in/faculty/mini" target="_blank">R. Sowdhamini</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
National Centre for Biological Sciences (NCBS), Bengaluru<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
November 17, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Protein-protein interactions are important for signal transduction and regulatory processes in biology. Amino acid mutations at key regions of such interactions could have strong consequences and hence implicated in diseases. Yet, such surfaces are hard to design drugs. I will describe few computational approaches to measure and understand protein-protein interfaces. Finally, I will describe how such objective measures could be useful by taking three examples.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">A three-player ecological system of pathogens</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/FakhtehGhanbarnejad_Nov19_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/FakhtehGhanbarnejad_Nov19_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/view/fakhtehgh" target="_blank">Fakhteh Ghanbarnejad</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Sharif University of Technology, Tehran, Iran<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
November 19, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
In ecological systems, heterogeneous interactions between pathogens take place simultaneously. This occurs, for instance, when two pathogens cooperate, while at the same time, multiple strains of these pathogens co-circulate and compete. Notable examples include the cooperation of human immunodeficiency virus with antibiotic-resistant and susceptible strains of tuberculosis or some respiratory infections with <em>Streptococcus pneumoniae</em> strains. Models focusing on competition or cooperation separately fail to describe how these concurrent interactions shape the epidemiology of such diseases. We studied this problem considering two cooperating pathogens, where one pathogen is further structured in two strains. The spreading follows a susceptible-infected-susceptible process and the strains differ in transmissibility and extent of cooperation with the other pathogen. We combined a mean-field stability analysis with stochastic simulations on networks considering both well-mixed and structured populations. We observed the emergence of a complex phase diagram, where the conditions for the less transmissible, but more cooperative strain to dominate are non-trivial, e.g. non-monotonic boundaries and bistability. Coupled with community structure, the presence of the cooperative pathogen enables the coexistence between strains by breaking the spatial symmetry and dynamically creating different ecological niches. These results shed light on ecological mechanisms that may impact the epidemiology of diseases of public health concern.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Adaptation in changing environments</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/KavitaJain_Nov24_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/KavitaJain_Nov24_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="http://old.jncasr.ac.in/jain/" target="_blank">Kavita Jain</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Jawaharlal Nehru Centre for Advanced Scientific Research (JNCASR), Bengaluru<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
November 24, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Natural environments are seldom static and therefore it is important to ask how a population adapts in a changing environment. I will consider a stochastically evolving population in which the mutant is beneficial during a part of the seasonal cycle and deleterious in another. The chance that the mutant spreads in the population depends on the instant it arose in the population and described by a time-inhomogeneous backward Fokker-Planck equation. I will discuss our analytical results for the first-passage probability and first-passage time, and their relevance to the evolution of genetic dominance.</font></h6></p>
	</div>
</div>
</div><br>

</div>

<div id="December2020"><br>

<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">The medley of stochastic heterogeneity in bacteria</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/Harapriya_Mohapatra_Dec8_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/Harapriya_Mohapatra_Dec8_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://harapriyamohapatra.wixsite.com/larv" target="_blank">Harapriya Mohapatra</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
National Institute of Science Education and Research (NISER), Bhubaneswar<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
December 8, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Pure bacterial cultures are a clonal population. Each individual cell in the population is genetically identical i.e isogenic. Thus, it is expected that the response of each individual cell of the population to external stimuli is the same, and at population level is synchronized and harmonious. However, recent studies reveal the existence of randomness in gene expressions at individual cell level. Such variability is observed even when the culture is maintained under constant external conditions. In my talk I will be discussing how this heterogeneity turns into a medley increasing the organism’s survival against the odds.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Repeat-aware methods for mapping and analyses of under-explored regions in the human genome</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/ChiragJain_Dec10_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/ChiragJain_Dec10_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/site/chirgjain/" target="_blank">Chirag Jain</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institute of Science (IISc), Bengaluru<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
December 10, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
The recent completion of human chromosomes X and 8 by the Telomere-to-Telomere Consortium has revealed highly repetitive satellite and segmentally duplicated sequences that were previously inaccessible to both de novo genome assembly and re-sequencing approaches. Over 10% of the current human reference cannot be reliably mapped with short sequencing reads, and this number will only grow as additional reference gaps are completed. Thus, the challenge will be to map reads and call variants within such repetitive, yet functionally important, regions of the genome. Long-read sequencing technologies hold promise, but the problem of accurately mapping long reads to complex genomic repeats must still be addressed. In this talk, I’ll highlight the fact that existing long read mappers often yield incorrect alignments and variant calls within long, near-identical repeats, as they remain vulnerable to allelic bias. In the presence of a non-reference allele within a repeat, a read sampled from that region could be mapped to an incorrect repeat copy because the standard pairwise sequence alignment scoring system penalizes true variants.To address the above problem, we propose a novel, long read mapping method that addresses allelic bias by making use of minimal confidently alignable substrings (MCASs). MCASs are formulated as minimal length substrings of a read that have unique alignments to a reference locus with sufficient mapping confidence (i.e., a mapping quality score above a user-specified threshold). This approach treats each read mapping as a collection of confident sub-alignments, which is more tolerant of structural variation and more sensitive to paralog-specific variants (PSVs) within repeats. We mathematically define MCASs and discuss an exact algorithm as well as a practical heuristic to compute them. The proposed method, referred to as Winnowmap2, is evaluated using simulated as well as real long read benchmarks using the recently completed gapless assemblies of human chromosomes X and 8 as a reference. We show that Winnowmap2 successfully addresses the issue of allelic bias, enabling more accurate downstream variant calls in repetitive sequences.</font></h6></p>
	</div>
</div>
</div><br>
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Conformational selection in a protein-protein interaction</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/KalyanChakrabarti_Dec17_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/KalyanChakrabarti_Dec17_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/site/chakrabartikalyans/" target="_blank">Kalyan Sundar Chakrabarti</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
KREA University, Andhra Pradesh<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
December 17, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Molecular recognition plays a central role in biology and protein dynamics has been acknowledged to be important in this process. However, it has been intensely debated for the last 50 years whether conformational changes happen before ligand binding to produce a binding-competent state (conformational selection) or are caused in response to ligand binding (induced fit) [1]. Proposals for both mechanisms in protein-protein interaction have been primarily based on structural arguments. However, the distinction between them is a question of the probabilities of going via these two opposing pathways. In this seminar I will discuss a direct demonstration of exclusive conformational selection in protein-protein recognition by measuring the flux for rhodopsin kinase binding to its regulator recoverin, an important molecular recognition in the vision system [2]. The combined use of nuclear magnetic resonance (NMR) spectroscopy, stopped-flow kinetics and isothermal calorimetry establishes that protein dynamics in free recover in limits the overall rate of binding.<br><br>
<ol>
<li>Changeux & Edelstein (2011) F1000 Biol. Rep. 3 19.</li>
<li>Chakrabarti et al. (2016) Cell Rep. 14 32-42.</li>
</font></h6></p>
</ol>
	</div>
</div>
</div><br>"
<div class="shadow p-3 mb-5 bg-white rounded">
	<center><b><h5><font color="#d0836e">Dispersal evolution: The Drosophila Story</font></h5></b></center><br>
<div class="row px-3">
	<div class="col-lg-3">
	<center><embed src="
https://www.imsc.res.in/~asamal/seminar/SutirthDey_Dec22_2020.pdf" width="200" height="265" name="plugin" type="application/pdf"><br>
	<a href="
https://www.imsc.res.in/~asamal/seminar/SutirthDey_Dec22_2020.pdf" target="_blank">Click to view the poster</a></center>
	</div><br>
	<div class="col-lg-9">
		<table>
		<tr><th class="align-top" style="width:75px" rowspan="2">Speaker: </th><td class="align-top"><font color="grey"><a href="https://sites.google.com/a/acads.iiserpune.ac.in/sdlab/pbl-iiser-p" target="_blank">Sutirth Dey</a></font></td></tr>
		<tr><td class="align-top"><font color="grey">
Indian Institutes of Science Education and Research (IISER), Pune<br></font></td></tr>
		<tr><th class="align-top" style="width:75px">Date:</th><td class="align-top"><font color="grey">
December 22, 2020</font></td></tr>
		<tr><th class="align-top" style="width:75px">Time:</th><td class="align-top"><font color="grey">
15:00 - 16:00</font></td></tr>
		</table><br>
	<b><h6>Abstract:</h6></b>
	<p><h6><font color="grey">
Given global climate change and large-scale degradation of ecosystems due to human activities, the continued survival of many species might depend on their ability to evolve to become better dispersers. Therefore, investigating the effects of dispersal evolution on natural populations is of considerable interest to ecologists and conservation biologists. Although dispersal is a complex multi-stage process, studies on dispersal evolution often investigate isolated components of dispersal like propensity (i.e. fraction of dispersers in the population) or ability (i.e. distance covered during dispersal). Thus, there is little understanding of how these components and their related costs interact during dispersal evolution and ultimately affect the dispersal kernel. To investigate these issues we subjected four replicate populations of Drosophila melanogaster to directional selection for increased dispersal, and compared them with matched controls. We found that the dispersal propensity and ability of the selected populations had increased simultaneously. Moreover, the selected populations had a greater frequency of long-distance dispersers (LDDs) and their dispersal kernels had evolved significantly greater standard deviation and reduced values of skew and kurtosis. In terms of life history, the dispersal selected populations had similar values of body size, fecundity and longevity as the controls. However, in terms of behavior, the selected populations evolved significantly greater locomotor activity, exploratory tendency, and aggression. These observations led to predictions about putative mechanisms that were confirmed through untargeted metabolomic fingerprinting using NMR spectroscopy. The selected flies had evolved greater amounts of glucose, AMP and NAD, suggesting elevated cellular respiration. At the same time, levels of neuropeptides related to aggression and exploration had increased.</font></h6></p>
	</div>
</div>
</div><br>

</div>


</div>
<!-- container section: end-->
<hr>
<div class="footer p-3">
<center><font color="grey">
 <h6>Copyright ©2021 All rights reserved</h6>
</font></center> 
</div>


    <script>
//Get the button
var mybutton = document.getElementById("myBtn");

// When the user scrolls down 20px from the top of the document, show the button
window.onscroll = function() {scrollFunction()};

function scrollFunction() {
  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
    mybutton.style.display = "block";
  } else {
    mybutton.style.display = "none";
  }
}

// When the user clicks on the button, scroll to the top of the document
function topFunction() {
  document.body.scrollTop = 0;
  document.documentElement.scrollTop = 0;
}


</script>


</body>
</html>
